styles/vars.scss

```diff
@@ -200,11 +200,16 @@ body {
   --xh-bg: var(--bg, white);
   --xh-bg-alt: var(--bg-alt, var(--xh-blue-gray-light));
   --xh-bg-highlight: var(--bg-highlight, #{mc-trans('blue', '200')});
+  --xh-selected-highlight: var(--selected-highlight, #{mc-trans('indigo', '300', 0.3)});
+  --xh-match-highlight: var(--match-highlight, #{mc-trans('amber', '700')});
```

Comment:
We will want to use more specific code-input scoped var names here, or maybe just not bother defining vars for these as long as we have something that looks reasonable. (Apps can always target the CSS classes and style without using vars.)

The issue with these is that they sound very general - if I were looking through our vars I might well assume the selected one applies to grid rows, or all text - or I wouldn't be quite sure what it did. 😄